packages/shared/lib/core/layer-2/actions/getSelectedAccountLayer2Tokens.ts

```diff
@@ -1,6 +1,6 @@
-import { getSelectedAccountWrappedNativeTokens } from '.'
+import { getSelectedAccountWrappedNativeTokens } from './getSelectedAccountWrappedNativeTokens'
 
-export async function getSelectedAccountLayer2Tokens(): Promise<{ amount: bigint; id: string }[]> {
+export async function getSelectedAccountLayer2Tokens(): Promise<unknown[]> {
```

Comment:
why? we know the return type  because of line 51-54 in `getSelectedAccountWrappedNativeTokens`